parser.c

```diff
@@ -190,14 +203,16 @@ static int parse_line(char *line, struct librole_graph *G)
     unsigned long len = strlen(line);
     unsigned long i = 0;
     char *last = line;
-    struct librole_ver role = {0, 0, 0, 10};
+    struct librole_ver default_role = {0, 0, 0, 10};
```

Comment:
Nitpick: this hunk makes diff more difficult to understand due to multiple `&role` -> `role` replacements. Perhaps it's better to skip it (since it has no functional changes)

Reply:
I tried to make as little line changes as possible but maybe it's better to change code as it has to be.